specification/resources/resource-manager/Microsoft.Authorization/stable/2019-01-01/policyDefinitions.json

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
In this file, one one property changed from enum to string. no other changes from 2018-05-01 api-version. Is that correct?

Reply:
Yes.